doc/source/developer_guide/dg_forcing.rst

```diff
@@ -0,0 +1,216 @@
+- The input data time axis is generally NOT read by the forcing subroutine.  The forcing frequency is hardwired into the model and the file record number is computed based on the forcing frequency and model time.  Mixing leap year input data and noleap model calendars (and vice versa) is not handled particularly gracefully.  The CICE model does not read or check against the input data time axis.
+- Data is read on the model grid, no spatial interpolation exists.
+- Data is often time interpolated linearly between two input timestamps to the model time each model timestep.
+
+In general, the following variables need to be defined by the forcing module,
+
+From Atmosphere:
+
+- zlvl    = atmosphere level height (m)
+- uatm    = model grid i-direction wind velocity component (m/s)
+- vatm    = model grid j-direction wind velocity component (m/s)
+- strax   = model grid i-direction wind stress (N/m^2)
+- stray   = model grid j-direction wind stress (N/m^2)
+- potT    = air potential temperature  (K)
+- Tair    = air temperature  (K)
+- Qa      = specific humidity (kg/kg)
+- rhoa    = air density (kg/m^3)
+- flw     = incoming longwave radiation (W/m^2)
+- fsw     = incoming shortwave radiation (W/m^2)
+- swvdr   = sw down, visible, direct  (W/m^2)
+- swvdf   = sw down, visible, diffuse (W/m^2)
+- swidr   = sw down, near IR, direct  (W/m^2)
+- swidf   = sw down, near IR, diffuse (W/m^2)
+- frain   = rainfall rate (kg/m^2 s)
+- fsnow   = snowfall rate (kg/m^2 s)
+
+From Ocean:
+
+- uocn    = ocean current, x-direction (m/s)
+- vocn    = ocean current, y-direction (m/s)
+- ss_tltx = sea surface slope, x-direction (m/m)
+- ss_tlty = sea surface slope, y-direction (m/m)
+- hwater  = water depth for basal stress calc (landfast ice)
+- sss     = sea surface salinity (ppt)
+- sst     = sea surface temperature (C)
+- frzmlt  = freezing/melting potential (W/m^2)
+- frzmlt_init= frzmlt used in current time step (W/m^2)
+- Tf      = freezing temperature (C)
+- qdp     = deep ocean heat flux (W/m^2), negative upward
+- hmix    = mixed layer depth (m)
+- daice_da= data assimilation concentration increment rate (concentration s-1)(only used in hadgem drivers)
+
+All variables have reasonable but static defaults and these will be used in ``default`` mode.
+
+To advance the forcing, the subroutines **get_forcing_atmo** and
+**get_forcing_ocn** are called each timestep from the step
+loop.  That subroutine computes the forcing year (``fyear``), calls the appropriate
+forcing data method, and then calls **prepare_forcing** which converts the 
+input data fields to model forcing fields.
+
+.. _JRA55forcing:
+
+JRA55 Atmosphere Forcing
+-------------------------
+
+The current default atmosphere forcing for gx3, gx1, and tx1 standalone grids for
+Consortium testing is the JRA55 forcing
+dataset :cite:`Tsujino18`.  The Consortium has released 5 years of forcing data, 
+2005-2009 for gx3, gx1, and tx1 grids.  Each year is a separate file and 
```

Comment:
At the moment, each grid has a separate file containing 5 years of data.

Reply:
Each year is a separate file.  Right now we tar up the 5 years of data onto zenodo for download.   I think that's what you're thinking about.

Reply:
yep, you're right